lib/tuya.js

```diff
@@ -731,6 +734,19 @@ const ZMAM02 = {
     },
 };
 
+const moesSwitch = {
+    powerOnBehavior: {
+        0: 'off',
+        1: 'on',
+        2: 'previous',
+    },
+    indicateLight: {
+        0: 'off',
+        1: 'switch',
+        2: 'position',
+    },
```

Comment:
Maybe not related, but I have a Moes switch that accept this posible values:
```
    off = 0x00
    light_when_on = 0x01
    light_when_off = 0x02
    freeze = 0x03
```
The freeze value keeps the current backlight state.

Reply:
I checked it in the tuya app and there are 3 values as it was defined.
<details><summary>screenshot</summary><img src="https://user-images.githubusercontent.com/16994749/152699117-9de41263-3bc2-41b4-afc7-eb1559ac593e.png" width="250" /></details>

But I tested the "freeze" value and it works. I'll create a new pr. Thanks!